specs/lib/removeUnusedVerticesSpec.js

```diff
@@ -435,9 +435,9 @@ describe('removeUnusedVertices', function() {
             2,2,2,
             2,2,2
         ];
-        var positionsBuffer = new Buffer(positions.length * 2);
+        var positionsBuffer = new Buffer(positions.length * 4);
         for (var i = 0; i < positions.length; i++) {
-            positionsBuffer.writeUInt16LE(positions[i], i * 2);
+            positionsBuffer.writeUInt16LE(positions[i], i * 4);

```

Comment:
`writeUInt16LE` should change to `writeFloatLE` to be consistent with the new change.